ui/cockpit/client/scripts/directives/change-version.html

```diff
@@ -0,0 +1,25 @@
+<form name="changeVersionForm" ng-if="isActive" class="change-version">
```

Comment:
Lets visualize an invalid ID
```suggestion
<form name="changeVersionForm" ng-if="isActive" class="change-version" ng-class="{'has-error': !isValid}">
```